docs/infrastructure/index.asciidoc

```diff
@@ -20,6 +20,11 @@ Kibana provides step-by-step instructions to help you add your data sources.
 The {infra-guide}[Infrastructure Monitoring Guide] is good source for more detailed
```

Comment:
I think you missed Dan's comment from earlier. This should say "is a good source".